packages/backend/src/controllers/moodle/handle.ts

```diff
@@ -24,6 +25,17 @@ export async function handleAssignments(courses: ICourse[], lastFetch: number):
         minute: 'numeric'
     };
 
+    return new Date(timestamp * 1000).toLocaleString('de-DE', dateOptions);
```

Comment:
```suggestion
    return new Date(timestamp * 1000).toLocaleString(config.langFormat || unknown, dateOptions);
```
Not an actual suggestion, but at one point in time we should think about i18n support for our backend

Reply:
Sure, feel free to start #126 #127 :yum: But yeah, you are right.